app_frontend.py

```diff
@@ -74,6 +76,19 @@ def not_found():
 
     return appli
 
+def get_config():
+    for loc in [ # search in the following locations:
+            os.curdir,
+            #os.path.expanduser("~"),
+            "/etc/inginious",
+            os.environ.get("INGINIOUS_CONF"),
+        ]:
+            filename = os.path.join(loc,"configuration.json")
+            if os.path.exists(filename):
+                return filename
+    raise "Cannot find configuration.json !"
+
 if __name__ == "__main__":
-    app = get_app("./configuration.json")
+    app = get_app(get_config())
+    os.chdir(os.path.dirname(inginious.__file__))
```

Comment:
This is a hack needed because we can only serve static files from the current directory.
But we want to serve static files installed with the package; hence the move.